PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -119,8 +97,47 @@ protected Control createDialogArea(Composite parent) {
         clusterNameField.setLayoutData(gridData);
         clusterNameField.setToolTipText("The HDInsight cluster name, such as 'mycluster' of cluster URL 'mycluster.azurehdinsight.net'.\n\n Press the F1 key or click the '?'(Help) button to get more details.");
 
+        Group clusterAccountGroup = new Group(container, SWT.NONE);
+        clusterAccountGroup.setText("The Cluster Account");
+        gridLayout = new GridLayout();
+        gridLayout.numColumns = 2;
+        clusterAccountGroup.setLayout(gridLayout);
+        gridData = new GridData();
+        gridData.horizontalSpan = 2;
+        gridData.widthHint = 350;
+        gridData.horizontalAlignment = SWT.FILL;
+        gridData.grabExcessHorizontalSpace = true;
+        clusterAccountGroup.setLayoutData(gridData);
+
+        userNameLabel = new Label(clusterAccountGroup, SWT.LEFT);
+        userNameLabel.setText("User Name:");
+        gridData = new GridData();
+        gridData.horizontalIndent = 38;
+        gridData.horizontalAlignment = SWT.RIGHT;
+        userNameLabel.setLayoutData(gridData);
+        userNameField = new Text(clusterAccountGroup, SWT.BORDER);
+        gridData = new GridData();
+        gridData.horizontalAlignment = SWT.FILL;
+        gridData.grabExcessHorizontalSpace = true;
+        userNameField.setLayoutData(gridData);
+        userNameField.setToolTipText("The user name of the HDInsight cluster.\n\n Press the F1 key or click the '?'(Help) button to get more details.");
+
+        passwordLabel = new Label(clusterAccountGroup, SWT.LEFT);
+        passwordLabel.setText("Password:");
+        gridData = new GridData();
+        gridData.horizontalIndent = 38;
+        gridData.horizontalAlignment = SWT.RIGHT;
+        passwordLabel.setLayoutData(gridData);
+        passwordField = new Text(clusterAccountGroup, SWT.PASSWORD | SWT.BORDER);
+        gridData = new GridData();
+        gridData.horizontalAlignment = SWT.FILL;
+        gridData.grabExcessHorizontalSpace = true;
+        passwordField.setLayoutData(gridData);
+        passwordField.setToolTipText("The password of the HDInsight cluster user provided.\n\n Press the F1 key or click the '?'(Help) button to get more details.");
+
         Group clusterStorageGroup = new Group(container, SWT.NONE);
-        clusterStorageGroup.setText("The Cluster Storage Information");
+        clusterStorageGroup.setText("The Cluster Storage Information (Optional)");
+        clusterStorageGroup.setToolTipText("The Cluster Storage Information provided can enable the Storage Explorer support.");
```

Comment:
>Storage Explorer support. [](start = 100, length = 25)

The Cluster Storage Information provided can enable the Storage Explorer support in the left tree view.

Reply:
Fix it.